### DISCUSSION TITLE
Gatsby Build not watching for WordPress changes

### DISCUSSION THREAD

**Discussion by jenishshrestha** [gatsbyjs/gatsby#29788] [Help]
Hi, I am new to Gatsby.

I built my first site on WordPress Gatsby and everything is great. When I run Gatsby develop, it works perfectly fine. Anything I change in WordPress gets reflected in Frontend after 4 sec (it is watching correctly).

But if run Gatsby build, it builds all the static pages but stops watching for WordPress changes.

Please guide me in this, what did I do wrong?

Thanks,

**Comment by rcapdepaula** [comment]
Maybe you need a webhook.

**Reply by jenishshrestha** [reply]
Yes, this was the solution. I used GitLab's webhook to trigger rebuild process in production but it feels like this is not really a satisfying method. I thought everything that happens in development happens in production as well. Gatsby keeps watch to WordPress changes and rebuilds the changes instantly.
